msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
+	cat <<-EOF
 
-test -z "$1" && {
-	echo "Usage: $0 <version> [optional components]"
-	exit 1
+Usage: msi/$me [-o <dir>] <version> [<optional-package>...]
```

Comment:
Maybe I'm wrong, but I thought $0 evaluated to the absolute path of the script (or the relative path from where the invocation is taking place)? This was an attempt to prevent that and just have the script name itself.